### ISSUE TITLE
Cannot load module in "multiple entry points" mode

### ISSUE THREAD

**Issue by ytkj** [s-panferov/awesome-typescript-loader#74]
Typescript 'external module' can not be loaded in 'multiple entry points' mode.
My config is below;
```javascript
module.exports = {
  context: path.join(__dirname, '/src'),
  resolve: { extensions: ['', 'ts'] },
  watch: false,
  module: {
    loaders: [{
      test: /\.ts$/,
      exclude: /node_modules/,
      loader: 'awesome-typescript-loader'
    }]
  },
  entry: {
    app1: './app1/app1.ts',
    app2: './app2/app2.ts'
  },
  output: {
    filename: '[name].js'
  }
}
```
and my gulpfile is below;
```javascript
var gulp = require('gulp');
var webpack = require('webpack-stream');
var path = require('path');

var sources = [
  'src/app1/app1.ts',
  'src/app2/app2.ts'
];

var option = require('./webpack.config');

gulp.task('build', function() {
  return gulp.src(sources)
    .pipe(webpack(option))
    .pipe(gulp.dest('dist'));
});

gulp.task('default', ['build']);
```
my project structure is;
```
gulpfile.js
src
    app1
        app1.ts
        sub1.ts
    app2
        app2.ts
        sub2.ts
dist
```
app1.ts;
```typescript
import { sub } from './sub1';
sub('Hello');
```
sub1.ts;
```typescript
export function sub(msg: string): void {
  console.log(msg + ' from sub1');
}
```
and output in console is below;
```shell
c:\dev\tmp>gulp
[18:08:21] Using gulpfile c:\dev\tmp\gulpfile.js
[18:08:21] Starting 'build'...
[18:08:24] Version: webpack 1.12.4
  Asset     Size  Chunks             Chunk Names
app1.js  1.63 kB       0  [emitted]  app1
app2.js  1.66 kB       1  [emitted]  app2

ERROR in ./src/app2/app2.ts
Module not found: Error: Cannot resolve 'file' or 'directory' ./sub2 in c:\dev\tmp\src\app2
 @ ./src/app2/app2.ts 1:13-30

ERROR in ./src/app1/app1.ts
Module not found: Error: Cannot resolve 'file' or 'directory' ./sub1 in c:\dev\tmp\src\app1
 @ ./src/app1/app1.ts 1:13-30
[18:08:24] Finished 'build' after 2.89 s
[18:08:24] Starting 'default'...
[18:08:24] Finished 'default' after 58 μs
```

I acturelly know that a possible solutions is using [shama/vinyl-named](https://github.com/shama/webpack-stream#multiple-entry-points), but I dont want to use this as much as possible.

**Comment by s-panferov** [comment]
@ytkj could you please try the latest version v0.15.0-rc.0?

**Comment by ytkj** [comment]
@s-panferov I'm using the version provided in #73, by @davidmiani. 

- node v5.0.0
- gulp v3.9.0
- webpack v1.12.4
- webpack

**Comment by aspirisen** [comment]
I have the same issue, `useCache: true` helps.
I use `"awesome-typescript-loader": "0.15.9"` and `"webpack": "1.12.9"`

**Comment by dietergoetelen** [comment]
In my case it can't find my typescript definition file. Is this related to the above issue or should i create a new issue?

**Comment by s-panferov** [comment]
This issue must be fixed by v0.15.10, please check
